src/components/command-bar/commands/search/unified-search/types.ts

```diff
@@ -0,0 +1,16 @@
+import type { Hit } from 'instantsearch.js'
+
+// TODO: set up an enum for this
+export type UnifiedSearchableContentType =
```

Comment:
Added this types file to hold some types I found myself re-using in a couple spots.